public/locations-list-map.js

```diff
@@ -618,14 +618,16 @@ function showMarkers(data, filters, showNearest) {
       }
     }
   }
+  var markerCluster = new MarkerClusterer(map, markers,
+    { imagePath: 'images/markercluster/m' });
```

Comment:
How is the markerCluster instance managed? showMarkers is called multiple times as the filters change. Need to tear down the previous cluster and create a new one?

Reply:
I think you can create it once when the map is created, then just call clearMarkers(), followed by addMarkers() as the filters change.